README.md

```diff
@@ -156,15 +156,92 @@ invocation queueing, allowing concurrent invocations.
 
 ### Info
 
-Not implemented yet.
+The info command shows the current state of the election:
```

Comment:
Do you plan on using the cobra docs generation for this stuff in the future? https://cobra.dev/#generating-documentation-for-your-command

Reply:
Good idea! just opened #18 to look into it. Tooling in general needs some effort, for example using goreleaser.